src/main/java/frc/robot/subsystems/instances/Magazine.java

```diff
@@ -25,21 +31,21 @@ public void periodic() {
 
   @Override
   public void run(double speed) {
-
+    magMotor.set(speed);
```

Comment:
why does it set speed?

Reply:
Well it's power more than speed, but I can fix that